packages/commercetools/composables/src/getters/storeGetters.ts

```diff
@@ -0,0 +1,101 @@
+import { UseStoreGetters, AgnosticAddress, AgnosticLocale } from '@vue-storefront/core';
+import { Store, Channel, Address } from '../types/GraphQL';
+import { StoresData, StoresItem } from '../types';
+import { FilterCriteriaRecord, Localized, filterArrayByCriteriaRecord } from '../helpers/internals';
+
+/**
+ * Types
+ */
+
+interface StoreFilterCriteria {
+  store?: FilterCriteriaRecord<Store>;
+  channel?: FilterCriteriaRecord<Channel>
+}
+
+type StoreChannelSetKeys =
+  'distributionChannels' |
+  'supplyChannels';
+
+/**
+ * Helpers
+ */
+
+function mapToAddress(address: Address): AgnosticAddress {
+  return {
+    addressLine1: `${address?.country ?? ''} ${address?.city ?? ''} ${address?.postalCode ?? ''}`.trim(),
+    addressLine2: `${address?.streetName ?? ''} ${address?.streetNumber ?? ''}`.trim(),
+    _address: address ?? null
+  };
+}
+
+function mapToLocale <T extends Localized>(localized: T): AgnosticLocale {
+  return {
+    code: localized.locale,
+    label: ''
```

Comment:
Should it be an empty string?

Reply:
CT GraphQL `LocalizedString` type does not provide locale label value.
The default value is empty string to match the `AgnosticLocale` type.
Maybe `AgnosticLocale["label"]` property should be optional?